lib/biran/configurinator.rb

```diff
@@ -53,8 +53,8 @@ def build_app_config
       }
 
       app_config.deep_merge! app_config_defaults
-      app_config[:secrets].deep_merge! get_secret_contents(app_config)
-      app_config[:db_config] = build_db_config
+      app_config[:secrets].deep_merge! get_secrets_content(app_config[:secrets_file_path])
+      app_config[:db_config].deep_merge! build_db_config
```

Comment:
Everything in this file is just a refactor of the process and using the new default values, which is done here.

Reply:
So the bug is that you were looking  in the wrong place for the values, as they where nested under app... the fix is to add them on the top level, as it makes more sense?

Reply:
Yes, the bug is we were looking in the wrong place. The values don't make sense nested under app which is the values used to do all the setup. Like file locations, files to generate, bindings, etc which shouldn't be passed down to the templates or have no need to be passed down. I don't know if the original plan was to use these default values here and merge them with the loaded values later and we forgot or assumed that was happening or ???
But, IMO, these values make more sense being in the top level since they are values we want to access and use in the templates.